ijroi/ijroi.py

```diff
@@ -119,7 +133,7 @@ def getfloat():
     points[:, 1] = [getc() for i in range(n_coordinates)]
     points[:, 0] = [getc() for i in range(n_coordinates)]
 
-    if options & SUB_PIXEL_RESOLUTION == 0:
+    if subPixelResolution == 0:
```

Comment:
This is a bool now / not an integer anymore, so use `if not subPixelResolution:` instead